src/method/method_callback.cpp

```diff
@@ -195,6 +204,14 @@ namespace cubmethod
 
     execute_info info = handler->execute (request);
 
+    /* register query_id for out resultset */
+    cubmethod::query_result *qresult = handler->get_current_result();
+    if (qresult->stmt_type == CUBRID_STMT_SELECT)
+      {
+	uint64_t qid = (uint64_t) info.qresult_infos[0].query_id;
```

Comment:
In info.qresult_infos[0], 0 means that it's not supported to handle the multi-statements.

Reply:
yes, it assumes that a statement can have only 1 result